formatter/src/formatter/element.rs

```diff
@@ -409,4 +433,255 @@ mod tests {
         </div>
         "#);
     }
+
+    // Self Closing Tag
```

Comment:
Since I forgot a specific use case in the first place, I copied over all the tests to be absolutely sure I did not forgot another one, but that feels fair yes